src/mui/list/InfiniteList.js

```diff
@@ -0,0 +1,216 @@
+                        <ViewTitle title={titleElement} />
+                        {actions &&
+                            React.cloneElement(actions, {
+                                resource,
+                                filters,
+                                filterValues,
+                                basePath,
+                                hasCreate,
+                                displayedFilters: this.state,
+                                showFilter: this.showFilter,
+                                theme,
+                                refresh: this.refresh,
+                            })}
+                    </div>
+                    {filters &&
+                        React.cloneElement(filters, {
+                            resource,
+                            hideFilter: this.hideFilter,
+                            filterValues,
+                            displayedFilters: this.state,
+                            setFilters: this.setFilters,
+                            context: 'form',
+                        })}
+                    <InfiniteScroll
+                        pageStart={1}
+                        initialLoad={true}
+                        loadMore={this.getNextPage.bind(this)}
+                        hasMore={this.props.hasMore}
+                        loader={loader}
+                    >
+                        {isLoading || total > 0 ? (
+                            <div key={version}>
+                                {children &&
+                                    React.cloneElement(children, {
+                                        resource,
+                                        ids,
+                                        data,
+                                        currentSort: {
+                                            field: query.sort,
+                                            order: query.order,
+                                        },
+                                        basePath,
+                                        isLoading,
+                                        setSort: this.setSort,
+                                    })}
+                            </div>
+                        ) : (
+                            <CardText style={styles.noResults}>
+                                {translate('aor.navigation.no_results')}
+                            </CardText>
+                        )}
+                    </InfiniteScroll>
+                </Card>
+            </div>
+        );
+    }
+}
+
+const getLocationSearch = props => props.location.search;
+const getQuery = createSelector(getLocationSearch, locationSearch => {
+    const query = parse(locationSearch);
+    if (query.filter && typeof query.filter === 'string') {
+        query.filter = JSON.parse(query.filter);
+    }
+    return query;
+});
+
+function mapStateToProps(state, props) {
+    const resourceState = state.admin.resources[props.resource];
```

Comment:
See my previous comment

Reply:
I trying this way but it doesn't work, I propose to add hasmorre props to the list and import it to infiniteList component